server/api/test/index.js

```diff
@@ -117,4 +117,24 @@ describe( 'api', function() {
 			.send( { username: 'validuser', password: 'validpassword' } )
 			.expect( 408, response, done );
 	} );
+
+	maybeIt( 'should return a 400 push verification request if the server supports it', function( done ) {
```

Comment:
I think we should avoid any config changes/renames here as these will likely need more changes to the desktop workflow, plus, these are out-of-scope for this PR.